src/client/components/TabPaneContent0/index.js

```diff
@@ -1,195 +1,185 @@
+import { Trans, useTranslation } from "react-i18next";
 import PropTypes from "prop-types";
 import React from "react";
-// import { useTranslation } from "react-i18next";
 
 function TabPaneContent0({ getTabLink }) {
-  // const { t } = useTranslation();
+  const { t } = useTranslation();
 
   return (
     <div>
       <div className="gray-box mt-0">
-        <h3>Tips for applying on UI Online</h3>
+        <h3>
+          <Trans t={t} i18nKey="tab0.header10" />
+        </h3>
         <ul>
           <li className="mb-3">
-            Everyone (UI and PUA applicants) uses the same form.{" "}
-            <a href="https://www.edd.ca.gov/Benefit_Programs_Online.htm">
-              Register or log in with Benefit Programs Online
-            </a>{" "}
-            and click “File New Claim.”
+            <Trans t={t} i18nKey="tab0.list10-item10">
+              {
+                // The text in this <Trans> must *approximately* match translation.json
+                // otherwise the link won't render.
+              }
+              Everyone (UI and PUA applicants) uses the same form.{" "}
+              <a href="https://www.edd.ca.gov/Benefit_Programs_Online.htm">
+                Register or log in with Benefit Programs Online
+              </a>{" "}
+              and click “File New Claim.”
+            </Trans>
           </li>
           <li className="mb-3">
-            File a new claim if you have already applied for UI but are eligible
-            for PUA because you are a business owner, independent contractor,
-            self-employed worker, freelancer, or gig worker affected by
-            COVID-19, <strong>and</strong> you have <strong>not</strong> already
-            appealed a $0 award notice, been asked to verify your identity, or
-            requested a wage investigation.
+            <Trans t={t} i18nKey="tab0.list10-item20" />
```

Comment:
Do we need to use the trans component here? Would it make sense to use the syntax we had been using previously?

Reply:
The only benefit of using the trans component over t() in our case is supporting inline <strong>/<i>/<p>/<br> tags, which I think we talked about and agreed is worth it?

Reply:
But in cases where the text doesn't have any tags, could we still use the old syntax?

Reply:
The old syntax feels slightly cleaner to me, but this is definitely not a deal-breaker for merging, so feel free to ignore!

Reply:
Oh, I see what you mean! Yes we could, and I agree the old syntax is cleaner, but I strongly prefer the consistency of "all content is in Trans tags in this file"-- it makes converting the files so much easier, too.

Reply:
Got it! Makes sense!